tests/test_controller.py

```diff
@@ -41,13 +46,26 @@ class ControllerWithDoMethods(salobj.Controller):
     command_names : `iterable` of `str`
         List of command names for which to make trivial ``do_<name>``
         methods.
+    allow_missing_callbacks : `bool`, optional
+        Allow missing ``do_<name>`` callback methods? Missing callback
+        methods will be replaced with one that raises salobj.ExpectedError.
+        This is intended for mock controllers, which may only support
+        a subset of commands.
+        Cannot be true unless ``do_callbacks`` is true.
```

Comment:
It looks that here, too, `do_callbacks` always is True so maybe remove this line from the comment?